torchbenchmark/models/dlrm/__init__.py

```diff
@@ -51,8 +51,9 @@ def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
         arch_mlp_bot = "512-512-64"
         arch_mlp_top = "1024-1024-1024-1"
         data_generation = "random"
-        mini_batch_size = 2048
-        num_batches = self.batch_size
+        mini_batch_size = self.DEFAULT_TRAIN_BSIZE if batch_size is None else batch_size
+        self.batch_size = mini_batch_size
+        num_batches = 1000
```

Comment:
Is there a reason we want to set `num_batches` to 1000 instead of 1?

Reply:
The paper uses num_batches=1000, so this was just extracted from there. Not sure how it affects the model itself here.

Reply:
Since we are creating a minimal reproduction of train/infer loop, I think it makes sense to only run 1 batch (or 3 batches at most, if 1 batch is less than 100 ms)